server/routes/routes.ts

```diff
@@ -0,0 +1,22 @@
+import KoaRouter from 'koa-router';
+import MovieService from '../services/movies.service';
+
+const router = new KoaRouter();
+
+router.get('/movies', ctx => {
+  ctx.body = { movies: MovieService.movies };
+});
+
+router.delete('/movies/delete/:id', ctx => {
+  ctx.body = { movies: MovieService.deleteMovie(Number(ctx.params.id)) };
+});
+
+router.post('/movies/add', ctx => {
+  ctx.body = { movies: MovieService.addMovie(ctx.request.body.movie) };
+});
+
+router.put('/movies/edit', ctx => {
+  ctx.body = { movies: MovieService.editMovie(ctx.request.body.movie) };
+})
```

Comment:
if you want to build your own server I'm ok with it, but it's behavior must be the same as the one that's given

Even with mocked data you've missed here params for GET movies like sorbBy, searchBy, pagination. Also server-side validation for add/edit movie.

So please, add all the missed features to the movies server, on the frontend side implement api calls for sorting/searching/select by categories, and lazy loading